src/main/java/org/maxandersen/starfish/Starfish.java

```diff
@@ -168,6 +169,16 @@ public static void editConfig()throws Exception{
 
 }
 
+//Function to Launch the Editor
+public static void launch_editor(Path directory,String ide,String final_clone_path)throws IOException, InterruptedException{
```

Comment:
since it is an editor we are assuming to start better to call the parameter `editor` instead of `ide`